.github/workflows/cron-scan-images-daily.yaml

```diff
@@ -19,8 +19,9 @@ jobs:
       - id: set-matrix
         name: Build image matrix
         run: |
-          OUTPUT=`./bin/scan-images/matrix.js`
-          echo "::set-output name=matrix::$OUTPUT"
+          echo "matrix<<EOF" >> "$GITHUB_OUTPUT"
```

Comment:
I think we have a lot of other places either : https://github.com/search?q=repo%3Areplicatedhq%2FkURL%20set-output&type=code

Reply:
this is an improvement. i am merging